pkg/config/config.go

```diff
@@ -278,3 +321,18 @@ func (c *Config) Connection() (connection *sdk.Connection, err error) {
 
 	return
 }
+
+// IsKeyringManaged returns the keyring name and a boolean indicating if the config is managed by the keyring.
+func IsKeyringManaged() (keyring string, ok bool) {
+	keyring = os.Getenv(properties.KeyringEnvKey)
+	return keyring, keyring != ""
+}
+
+// Returns the available keyrings on the current host
+func GetKeyrings() []string {
```

Comment:
nit: linter suggestion: Comment should have the following format 'GetKeyrings ...'

Reply:
updated